src/app/pages/news/11.3/04-notificationApi.md

```diff
@@ -1,6 +1,11 @@
 ## Business Notification API {#notificationAPI}
 
-... notifications
+Just introduced in 11.2, we now extended the Notifications feature even more.
```

Comment:
This introduction is hard to read. Trying to simplify it. Feel free to keep it as is.

> The Notification feature introduced with 11.2, received further extensions, to address issues reported by our precious early bird user base.